src/System.Net.Security/tests/StressTests/SslStress/Utils/ChecksumHelpers.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace SslStress.Utils
+{
+    // Adapted from https://github.com/dotnet/corefx/blob/41cd99d051102be4ed83f4f9105ae9e73aa48b7c/src/Common/tests/System/IO/Compression/CRC.cs
+    public static class CRC
```

Comment:
What adaptation was needed?  Why can't we just have a single shared implementation?

Reply:
In this case spanification. Note that this comment was copied from the stress project which also added a couple of string methods.

Reply:
Can we make the changes to the shared implementation and use the spanified version in the original source as well?

Reply:
Yes, but it's unlikely I'll get this done before the cutoff.